server/views/temporary-accommodation/assessments/partials/_archivedAssessmentsTable.njk

```diff
@@ -0,0 +1,39 @@
+{{ govukTable({
+                firstCellIsHeader: true,
+                attributes: {
+                'data-module': 'moj-sortable-table'
+                },
+                head: [
```

Comment:
We end up not using `archivedTableHeaders`. Can it be deleted from the controller?